### ISSUE TITLE
[Feature] Multiple sync source in configuration

### ISSUE THREAD

**Issue by zhongkairen** [zhongkairen/airtable-sync-js#49]
option1
Same `config.json` file, having multiple configurations.

option 1a
an array of the same configuration

option 1b
an array of github configuration, a shared configuration of airtable target base.

option 2
Same `.config` folder, having different json configuration files.

option 2 is preferred as per separation of concerns.
Error reporting, we can iterate all the json files in the folder and run separately for each file.
Just add a switch -c for specifying configuration file path.

**Comment by zhongkairen** [comment]
The fields are empty on Airtable base, but filled in GitHub.

```
npm run start --  --verbose  --config .config/airtable-sync-js.json
2024-11-29T21:49:25.897Z info: [src/airtable-sync.js:163] synced 1 record(s): no changes.
```

```json
"fieldMap": {
      "Starto": "Start Date",
      "Endo": "Delivery Date"
    },
```

**Comment by zhongkairen** [comment]
released in v1.1.0.
